From 5ded9661e20ea7fa1e867e606b65fa0a3ecce695 Mon Sep 17 00:00:00 2001
From: Arulselvan M <arulselvan.m@intel.com>
Date: Wed, 21 Nov 2012 13:56:49 +0530
Subject: [PORT FROM MAIN] Enable WMA9 audio Decoding support

BZ: 50306

This patch contains changes in the libstagefright
to enable audio decoding capabilities

Change-Id: I3861958c7142795dc68829117a1751a7fad113d3
Orig-Change-Id: Ie4058be181b7bdfe1c22daa0a8c9139d8a1e403b
Signed-off-by: Arulselvan M <arulselvan.m@intel.com>
Reviewed-on: http://android.intel.com:8080/75141
Reviewed-by: lab_cactus <lab_cactus@intel.com>
Reviewed-by: Liang, Dan <dan.liang@intel.com>
Tested-by: Tong, BoX <box.tong@intel.com>
Reviewed-by: Tong, BoX <box.tong@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
---
 include/media/stagefright/MetaData.h |    6 ++++++
 1 files changed, 6 insertions(+), 0 deletions(-)

diff --git a/include/media/stagefright/MetaData.h b/include/media/stagefright/MetaData.h
index 9ab3edc..b228960 100644
--- a/include/media/stagefright/MetaData.h
+++ b/include/media/stagefright/MetaData.h
@@ -161,6 +161,12 @@ enum {
     kKeyCryptoDefaultIVSize = 'cryS',  // int32_t
 
     kKeyPssh              = 'pssh',  // raw data
+#ifdef USE_INTEL_ASF_EXTRACTOR
+    kKeyWmaBlockAlign     = 'blka',
+    kKeyWmaFormatTag      = 'frmt',
+    kKeyBitPerSample      = 'btps',
+#endif //ifdef USE_INTEL_ASF_EXTRACTOR
+
 };
 
 enum {
-- 
1.7.4.1

